release/api/v1alpha1/bundle_types.go

```diff
@@ -196,6 +197,15 @@ type DockerBundle struct {
 	Metadata        Manifest `json:"metadata"`
 }
 
+type CloudStackBundle struct {
+	Version              string   `json:"version"`
+	ClusterAPIController Image    `json:"clusterAPIController"`
+	KubeProxy            Image    `json:"kubeProxy"`
+	Manager              Image    `json:"manager"`
```

Comment:
I think we don't need the `Manager` here. We only have the ClusterAPIController. For vSphere we had the `cloud-provider-vsphere` which was the cloud-controller-manager component.

Reply:
Got it, will remove!